src/Workspaces/Core/Portable/CodeStyle/CodeStyleOptions.cs

```diff
@@ -4,430 +4,35 @@
 
 #nullable enable
 
-using System.Collections.Immutable;
-using System.Diagnostics;
-using Roslyn.Utilities;
-using static Microsoft.CodeAnalysis.CodeStyle.CodeStyleHelpers;
-
-#if CODE_STYLE
-namespace Microsoft.CodeAnalysis.Internal.Options
-#else
 using Microsoft.CodeAnalysis.Options;
+
 namespace Microsoft.CodeAnalysis.CodeStyle
-#endif
 {
+    /// <inheritdoc cref="CodeStyleOptions2"/>
     public class CodeStyleOptions
```

Comment:
This type now just has simple public fields that forward to `CodeStyleOptions2`. All the deleted code below is just moved to `CodeStyleOptions2` without any semantic changes.